modules/update-cluster-version-object.adoc

```diff
@@ -37,32 +37,46 @@ $ oc adm upgrade --include-not-recommended
 +
 [NOTE]
 ====
-The additional `--include-not-recommended` parameter includes updates that are available but not recommended due to a known risk that applies to the cluster.
+The additional `--include-not-recommended` parameter includes updates that are available with known issues that apply to the cluster.
 ====
 +
 .Example output
 [source,terminal]
 ----
-Cluster version is 4.10.22
+$ oc adm upgrade --include-not-recommended
+Cluster version is 4.13.40
 
-Upstream is unset, so the cluster will use an appropriate default.
-Channel: fast-4.11 (available channels: candidate-4.10, candidate-4.11, eus-4.10, fast-4.10, fast-4.11, stable-4.10)
+Upstream: https://api.integration.openshift.com/api/upgrades_info/graph
```

Comment:
Sorry, this is an artifact of [my ClusterBot testing][1].  Can we leave this line at the old:

```
Upstream is unset, so the cluster will use an appropriate default.
```

which is still what modern `oc` will say when the `upstream` property is empty.

[1]: https://github.com/openshift/openshift-docs/pull/74472#discussion_r1630384846

Reply:
Can do!